Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -242,73 +242,177 @@ private T CallPlexHelper<T>(Func<PlexHelper, T> act)
         }
 
         #region plexapi
-        public class PlexEventSuper
-        {
-            public string user { get; set; }
-            public string Owner { get; set; }
-        }
-#pragma warning disable 0649
+        #pragma warning disable 0649
+
         public class PlexEvent
         {
             [Required]
+            [DataMember(Name = "event")]
             public string Event;
+
+            [DataMember(Name = "user")]
             public bool User;
+
+            [DataMember(Name = "owner")]
             public bool Owner;
 
             [Required]
+            [DataMember(Name = "Account")]
             public PlexAccount Account;
+
             [Required]
+            [DataMember(Name = "Server")]
             public PlexBasicInfo Server;
+
+            [DataMember(Name = "Player")]
             public PlexPlayerInfo Player;
+
+            [DataMember(Name = "Metadata")]
             [Required]
             public PlexMetadata Metadata;
 
             public class PlexAccount
             {
+                [DataMember(Name = "id")]
                 public int Id;
-                public string Thumb;
+
+                [DataMember(Name = "thumb")]
+                public string Thumbnail;
+
+                [DataMember(Name = "title")]
                 public string Title;
             }
 
             public class PlexBasicInfo
             {
+                [DataMember(Name = "title")]
                 public string Title;
+
+                [DataMember(Name = "uuid")]
                 public string Uuid;
             }
 
             public class PlexPlayerInfo : PlexBasicInfo
             {
+                [DataMember(Name = "local")]
                 public bool Local;
-                public string PublicAddress;
+
+                [DataMember(Name = "publicAddress")]
+                public string Adress;
```

Comment:
This I would leave named as PublicAddress, since even when playing local it'll show the external IP.